README.md

```diff
@@ -1,16 +1,49 @@
 # README
 
-TypeScript implementation for LIonWeb standards - currently: the `lioncore` meta-metamodel in [`src/m3/types.ts`](./src/m3/types.ts).
+TypeScript implementation for LIonWeb standards - currently: the LIonCore meta-metamodel (M3) in [`src/m3/types.ts`](src/m3/types.ts).
```

Comment:
I think that `LionCore` would be easier to write and nicer to read, compared to `LIonCore`

Reply:
Hmmm, not sure I would like to loose the consistency...

Reply:
Maybe we should make an issue of this in the `organization` repo? It seems like this is an “overall style guide” kind-of-thing.